README.md

```diff
@@ -21,13 +21,12 @@ For a gentle introduction, please take a look at our [Quickstart Guide](https://
 
 - [Quickstart](https://astronomer.github.io/dag-factory/latest/getting-started/quick-start-astro-cli/)
 - [Benefits](#benefits)
-- [Features](#features)
-  - [Dynamically Mapped Tasks](https://astronomer.github.io/dag-factory/latest/features/dynamic_tasks/)
-  - [Multiple Configuration Files](#multiple-configuration-files)
-  - [Callbacks](#callbacks)
-  - [Custom Operators](#custom-operators)
-- [Notes](#notes)
-  - [HttpSensor (since 1.0.0)](#httpsensor-since-100)
+- [Features](https://astronomer.github.io/dag-factory/latest/features/dynamic_tasks/)
+    - [Dynamically Mapped Tasks](https://astronomer.github.io/dag-factory/latest/features/dynamic_tasks/)
+    - [Multiple Configuration Files](https://astronomer.github.io/dag-factory/latest/features/dynamic_tasks/)
+    - [Callbacks](https://astronomer.github.io/dag-factory/latest/features/dynamic_tasks/)
+    - [Custom Operators](https://astronomer.github.io/dag-factory/latest/features/dynamic_tasks/)
+    - [HttpSensor](https://astronomer.github.io/dag-factory/latest/features/dynamic_tasks/)
```

Comment:
Can we reconfirm all the linked URLs here, they appear all to be pointing to dynamic_tasks

Reply:
Aha, I forgot to submit the git push. Just did it now, thanks!